core/pom.xml

```diff
@@ -53,9 +45,15 @@
 
 		<!-- for log -->
 		<dependency>
```

Comment:
In general you want to only include the slf4j api, not the implementation. I've left the implementation for tests - http://slf4j.org/faq.html#maven2.